pulumi/grapl/__main__.py

```diff
@@ -368,11 +369,21 @@ def main() -> None:
             port=5532,
         )
 
+        uid_allocator_db = LocalPostgresInstance(
+            name="uid-allocator-db",
+            port=5532,
+        )
+
         pulumi.export("plugin-work-queue-db-hostname", plugin_work_queue_db.hostname)
         pulumi.export("plugin-work-queue-db-port", str(plugin_work_queue_db.port))
         pulumi.export("plugin-work-queue-db-username", plugin_work_queue_db.username)
         pulumi.export("plugin-work-queue-db-password", plugin_work_queue_db.password)
 
+        pulumi.export("uid-allocator-db-hostname", uid_allocator_db.hostname)
+        pulumi.export("uid-allocator-db-port", str(uid_allocator_db.port))
+        pulumi.export("uid-allocator-db-username", uid_allocator_db.username)
+        pulumi.export("uid-allocator-db-password", uid_allocator_db.password)
+
```

Comment:
You don't need to export these unless they're used in integration tests (they're not)

Reply:
They will be though post base.